vector/src/main/java/im/vector/app/core/ui/views/ShieldImageView.kt

```diff
@@ -36,21 +36,30 @@ class ShieldImageView @JvmOverloads constructor(
         }
     }
 
-    fun render(roomEncryptionTrustLevel: RoomEncryptionTrustLevel?) {
+    fun render(roomEncryptionTrustLevel: RoomEncryptionTrustLevel?, borderLess: Boolean = false) {
         isVisible = roomEncryptionTrustLevel != null
 
         when (roomEncryptionTrustLevel) {
             RoomEncryptionTrustLevel.Default -> {
                 contentDescription = context.getString(R.string.a11y_trust_level_default)
-                setImageResource(R.drawable.ic_shield_black)
+                setImageResource(
```

Comment:
@chagai95 updating these to use [setAttributeTintedImageResource](https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/core/extensions/ViewExtensions.kt#L73) and then supplying a theme colour will allow customisations